src/geo/lng_lat.js

```diff
@@ -12,10 +12,9 @@ export const earthRadius = 6371008.8;
 
 /**
  * A `LngLat` object represents a given longitude and latitude coordinate, measured in degrees.
- * These coordinates are based on the [WGS84 (EPSG:4326) standard](https://en.wikipedia.org/wiki/World_Geodetic_System#WGS84).
- *
- * Mapbox GL uses longitude, latitude coordinate order (as opposed to latitude, longitude) to match the
- * [GeoJSON specification](https://tools.ietf.org/html/rfc7946).
+ * These coordinates use longitude, latitude coordinate order (as opposed to latitude, longitude)
+ * to match the [GeoJSON specification](https://datatracker.ietf.org/doc/html/rfc7946#section-4),
+ * which is equivalent to the [OGC:CRS84 coordinate reference system](https://datatracker.ietf.org/doc/html/rfc7946#section-4).
```

Comment:
Nit: Do we need the same link twice?

Reply:
@SnailBones great catch, removed 2nd link in 54201b8d3 👍🏼